tools/releaseBuild/Images/GenericLinuxFiles/PowerShellPackage.ps1

```diff
@@ -41,17 +44,20 @@ finally
 }
 
 $linuxPackages = Get-ChildItem "$location/powershell*" -Include *.deb,*.rpm
-    
-foreach($linuxPackage in $linuxPackages) 
-{ 
+
+foreach ($linuxPackage in $linuxPackages)
+{
     Copy-Item -Path $linuxPackage.FullName -Destination $destination -force
 }
 
-if($AppImage.IsPresent)
+$extraPackages = @()
+switch ($ExtraPackage)
 {
-    $appImages = Get-ChildItem -Path $location -Filter '*.AppImage'
-    foreach($appImageFile in $appImages) 
-    { 
-        Copy-Item -Path $appImageFile.FullName -Destination $destination -force
-    }
+    "AppImage" { $extraPackages += @(Get-ChildItem -Path $location -Filter '*.AppImage') }
```

Comment:
Instead of get and then copy, can't we just copy?

```PowerShell
Copy-Item -Path "$location/*.AppImage", "$location/*.tar.gz" -Destination $destination -Force
```

Reply:
Or even this:

```Powershell
Copy-Item -Path "$location/powershell*/*.deb","$location/powershell*/*.rpm","$location/*.AppImage", "$location/*.tar.gz" -Destination $destination -Force

Reply:
Please make sure the code logs all copies.

Reply:
@TravisEz13 Is `Write-Verbose -verbose` sufficient for the logging purpose?

Reply:
Thanks @adityapatwardhan, your comment is addressed.
@TravisEz13 I added `Write-Verbose -Verbose` for each file to be copied.